### PR TITLE
fix /_api/cluster/endpoints URL for active failover

### PR DESCRIPTION
**Pull request by jsteemann**

### CODE REVIEW

**Comment by jsteemann** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/3630/

File: arangod/Cluster/RestClusterHandler.cpp [hunk 1]
```diff
@@ -42,32 +42,41 @@ RestClusterHandler::RestClusterHandler(GeneralRequest* request, GeneralResponse*
...
 RestStatus RestClusterHandler::execute() {
-  if (_request->requestType() != RequestType::GET) {
-    generateError(rest::ResponseCode::METHOD_NOT_ALLOWED, TRI_ERROR_HTTP_METHOD_NOT_ALLOWED,
-                  "only the GET method is allowed");
-    return RestStatus::DONE;
-  }
+  ServerState::RoleEnum role = ServerState::instance()->getRole();
+
+  if (ServerState::instance()->isCoordinator(role) ||
+      (ServerState::instance()->isSingleServer(role) && AgencyCommManager::isEnabled())) {
```
Comment (graetzer) [inline arangod/Cluster/RestClusterHandler.cpp]: doesn't handleCommandEndpoints check the same conditions ?
Also this handler is only registered if the cluster feature is enabled

Comment (jsteemann) [reply arangod/Cluster/RestClusterHandler.cpp]: However, the RestHandler is still callable in active failover mode. That I verified.

Comment (graetzer) [reply arangod/Cluster/RestClusterHandler.cpp]: but it should not be callable in single server mode

**Comment by jsteemann** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/3641/

**Pull request merged**